kevinb5617/ResuMake
# Create eslint action

# Description

The project should have an ESLint configuration file and associated GitHub action to run it on all commits/pull requests.

# Related Issue(s) and Pull Request(s)

#16

# Solution

An `.eslintrc.js` file was added to the repository (specifically the `app/` directory), and a GitHub action for running it was created.

# Steps to Test

(Assuming this repository is the "upstream" remote)
1. `git fetch upstream`
2. `git checkout create-eslint-action`
3. `cd app`
4. `npm install`
5. `npx eslint .eslintrc.js --ext .js,.jsx .`
6. You should see no output, since all the linting errors are fixed.
7. Open `src/App.js` and add an extra blank line at the end of the file.
8. Run step 5 again and you should see an error.
9. `npx eslint .eslintrc.js --ext .js,.jsx . --fix`
10. The extra line should now be removed.
11. You can also run the GitHub action on this branch to verify no linting errors.

# Known Issues

NONE

# Checklist
